radiomics/imageoperations.py

```diff
@@ -248,7 +239,117 @@ def applyThreshold(inputImage, lowerThreshold, upperThreshold, insideValue=None,
   return tif.Execute(inputImage)
 
 
-def swt3(inputImage, wavelet="coif1", level=1, start_level=0):
+def applyFilterOriginal(inputImage, **kwargs):
```

Comment:
I can change the signature of the name quite easily, but it'll have to be a signature in common for all filters and the original image. How about "get<filtername>Image"? This doesn't specify filter and as a bonus is quite like how we define the feature function signature...